kurtosis-devnet/interop.yaml

```diff
@@ -47,7 +47,7 @@ optimism_package:
         node0: &x-node
           el:
             type: op-geth
-            image: ""
+            image: "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:710a9697279600e7812c8847283eeb750d60ed8e@sha256:0e0dd748d5596ebd728b8d4691f26c4a4a6f387a834f2aa38534e192514bec2a"
```

Comment:
This is necessary because interop implies jovian, so we need a jovian-compatible op-geth to run this network with. With this image running, I manually confirmed the 17 byte extra data shows up on both network ELs.